src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -86,14 +82,14 @@ const styles = StyleSheet.create({
   },
   input: {
     paddingStart: 10,
-    lineHeight: 36,
+    lineHeight: 28,
     flex: 1,
     paddingBottom: 0,
     paddingTop: 0,
     fontFamily: fonts.inter[400],
     fontSize: 14,
     alignSelf: 'center',
-    color: COLORS.blackgrey
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The line height, color, and padding of the input field have been changed. Make sure these changes improve readability and do not cause any layout issues.

```diff
- lineHeight: 36,
- color: COLORS.blackgrey
+ lineHeight: 28,
+ color: COLORS.gray400
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->